cluster/addons/fluentd-gcp/fluentd-gcp-image/Dockerfile

```diff
@@ -30,17 +30,22 @@ ENV DO_NOT_INSTALL_CATCH_ALL_CONFIG true
 
 RUN apt-get -q update && \
     apt-get install -y curl && \
+    apt-get install -y gcc && \
+    apt-get install -y make && \
     apt-get clean && \
     curl -s https://dl.google.com/cloudagents/install-logging-agent.sh | bash
 
 # Install the record reformer plugin.
 RUN /usr/sbin/google-fluentd-gem install fluent-plugin-record-reformer
 
+# Install plugin to read systemd-journal
+RUN /usr/sbin/google-fluentd-gem install fluent-plugin-systemd -v 0.0.3
+
 # Remove the misleading log file that gets generated when the agent is installed
 RUN rm -rf /var/log/google-fluentd
 
 # Copy the Fluentd configuration file for logging Docker container logs.
 COPY google-fluentd.conf /etc/google-fluentd/google-fluentd.conf
 
 # Start Fluentd to pick up our config that watches Docker container logs.
-CMD /usr/sbin/google-fluentd "$FLUENTD_ARGS"
+CMD /bin/sh -c "rm /lib/x86_64-linux-gnu/libsystemd* && cp /host/lib/libsystemd* /lib/x86_64-linux-gnu/ && /usr/sbin/google-fluentd \"$FLUENTD_ARGS\""
```

Comment:
This'll break on deployments where we don't mount /host/lib. If we plan to maintain separate images, then we should have both of them checked in, not overwrite the old one with this one, although it would seem as though we could get around needing separate images if we just didn't define a CMD in the image (and defined it in the yaml instead).